index.html

```diff
@@ -217,6 +217,28 @@ <h3 class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900">Skills</h3>
 			</li>
 		</ul>
 	  </section>
+	  <section class="contact">
+		  <div class="contact__container">
+			<h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
+			<p class="contact__dsc fw-500 fz-20 lh-28">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+			<form action="https://formspree.io/f/xqkwbzdb" method="post">
```

Comment:
- [ ] **Your form element is missing the background image that is present in the design template. Please add it**

Form in Design Template 
:--------------------------------:
![image](https://user-images.githubusercontent.com/59168430/121258023-03f1e480-c8cc-11eb-9e9c-0c59de5a5dd4.png)

It is a very important skill to convert a given design into a functional UI. It will keep on getting better more you practice it.